eng/pipelines/dotnet-monitor-publish.yml

```diff
@@ -0,0 +1,55 @@
+trigger: none
+pr: none
+
+parameters:
+- name: IsTestRun
+  type: boolean
+  default: true
+- name: IsDryRun
+  type: boolean
+  default: true
+
+variables:
+- template: /eng/common/templates-official/variables/pool-providers.yml@self
+- name: _TeamName
+  value: DotNetCore
+  readonly: true
+- group: Release-Pipeline
+- name: IsDryRun
+  value: ${{ parameters.IsDryRun }}
+  readonly: true
+- name: IsTestRun
+  value: ${{ parameters.IsTestRun }}
+  readonly: true
+
+resources:
+  pipelines:
+  - pipeline: dotnet-monitor_build
+    project: internal
```

Comment:
This pipeline is already in the internal project, so setting this is not necessary.